src/main/java/cubicchunks/worldgen/generator/custom/CustomFeatureProcessor.java

```diff
@@ -67,9 +67,6 @@ public CustomFeatureProcessor(GeneratorStage generatorStage, String name, ICubeC
 	@Override
 	public Set<Cube> calculate(Cube cube) {
 
-		if (!this.canGenerate(cube)) {
```

Comment:
I'm almost sure that after removing this, the canGenerate method is unused and can be removed.